includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -151,17 +151,30 @@ public function need_update_idempotency_key( $source_object, $error ) {
 		);
 	}
 
+	/**
+	 * Checks if keys are set.
```

Comment:
Please expand the comment to note that it returns false if a secret key was set for the publishable key.

Reply:
Added in 3d23123